src/aat/resources/features/F-036/S-199.td.json

```diff
@@ -29,6 +29,7 @@
         "case_type_id": "AAT_AUTH_15",
         "created_date": "[[ANY]]",
         "last_modified": "[[ANY]]",
+        "last_state_modified_date": "[[ANY]]",
```

Comment:
not that great that we have to update so many test data files when we add a property to the json. This number of json files will only grow in the future. I see some serious maintainability problems @MSancaktutar

Reply:
If the outcomes of many functions are changing out of a code change, we should update all the functional verifications. That's integral part of the delivery. However, to reduce the maintenance efforts, we already have data inheritance/re-use mechanisms in place. The current use of the mechanism is already reducing that load a huge amount. If further uses are possible, we can improve the test data, just as with anything else.

Reply:
For example, SB upgrade brought a new Strict-Transport-Security header to almost all our responses, and we added it to only 2 or 3 places among 300+ files, to reflect the change to the verification of hundreds of test scenarios. This is how things should ideally work, and we can achieve it gradually improving the quality of test data files for smarter use of the framework etc.

Reply:
adding a new property to the case data it's something that it's gonna happen again in the future. I'd suggest we apply the inheritance/re-use mechanism for these cases  as well to reduce the number of test files we'll have to update in the future. Too many test files had to be updated in this PR. I'd suggest we raise an Improvement ticket for this